modules/azure/sql.go

```diff
@@ -0,0 +1,211 @@
+package azure
```

Comment:
NIT: I mentioned this in another PR ([this comment](https://github.com/gruntwork-io/terratest/pull/641#discussion_r498890874)), but it feels extraneous to provide getter functions that go directly to the attribute of an object, and instead we should provide a single function that returns the object.

For example, instead of offering `GetSQLServerID` and `GetSQLServerFullDomainName`, we could offer `GetSQLServer` which returns the object and one can access the `ID` and `FullyQualifiedDomainName` fields on the same object.

Reply:
fixed, created two getter one for sqlserver and one for sqldb and removed the properties getters